### PR TITLE
Allow NaN to be handled by parsers without getting into an infinite l…

### PR DESCRIPTION
**Pull request by dchambers**
…oop.

### CODE REVIEW

File: brjs-sdk/sdk/libs/javascript/br-ie8-shim/ie8-shim.js [hunk 1]
```diff
@@ -47,3 +47,9 @@ if (Function.prototype.bind && typeof console == "object" && typeof console.log
 		console[method] = Function.prototype.call.bind(console[method], console);
 	}
 }
+
+if(!Number.isNaN) {
+	Number.isNaN = Number.isNaN || function(value) {
+	    return typeof value === "number" && value !== value;
```
Comment (andyberry88) [inline brjs-sdk/sdk/libs/javascript/br-ie8-shim/ie8-shim.js]: @dchambers Out of interest why didn't you use `isNan()` (https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/isNaN) here?

Comment (dchambers) [inline brjs-sdk/sdk/libs/javascript/br-ie8-shim/ie8-shim.js]: `isNaN()` only produces sensible results for numbers, whereas `Number.isNaN()` accepts argument of any type.

Comment (andyberry88) [inline brjs-sdk/sdk/libs/javascript/br-ie8-shim/ie8-shim.js]: :+1:

**Comment by thecapdan** [comment]
tested. fixed

**Pull request merged**